### PR TITLE
Set the destination incentive addresses on package processing

### PR DESCRIPTION
**Pull request by reednaa**
By setting the escrow, we can let plugins get more access to context for a message. This is better than optimistically getting the destination.

### CODE REVIEW

**Comment by reednaa** [comment]
@jsanmigimeno Can you verify that this doesn't break anything? It is needed for the explorer.

File: src/collector/mock/mock.service.ts [hunk 1]
```diff
@@ -155,6 +155,12 @@ const bootstrap = async () => {
           // It can be used by plugins to facilitate other jobs.
           await store.setAmb(amb, messageEvent.transactionHash);
 
+          // Set destination address for the bounty.
+          store.registerDestinationAddress({
```
Comment (jsanmigimeno) [inline src/collector/mock/mock.service.ts]: Needs awaiting (or if this is intentional prepend the call with 'void')

File: src/collector/wormhole/wormhole.service.ts [hunk 2]
```diff
@@ -88,6 +97,22 @@ const bootstrap = async () => {
...
+
+        // Decode payload
+        const decodedPayload = ParsePayload(amb.payload);
+        if (decodedPayload === undefined) {
+          logger.info('Could not decode payload.');
+          continue;
+        }
+
+        // Set destination address for the bounty.
+        store.registerDestinationAddress({
```
Comment (jsanmigimeno) [inline src/collector/wormhole/wormhole.service.ts]: Needs awaiting (or if this is intentional prepend the call with 'void')

**Review by jsanmigimeno — CHANGES_REQUESTED** [review]
Looks good, just 2 minor comments

**Pull request merged**